openshift/manifests/14-clustercatalog-catalogd-openshift-certified-operators.yml

```
@@ -0,0 +1,11 @@
+---
+apiVersion: olm.operatorframework.io/v1alpha1
+kind: ClusterCatalog
+metadata:
+  name: catalogd-openshift-certified-operators
+spec:
+  source:
+    image:
+      pollInterval: 10m
```

Something we noticed in Bryce's dev work on cluster-olm-operator that we needed to change.

```suggestion
      pollInterval: 10m0s
```